docs/DesignDocs/CustomAttestation.md

```diff
@@ -927,9 +952,24 @@ size_t params_size = sizeof(params);
 oe_claim_t claims = { ... };
 size_t claims_size = ...;
 
+/* Receive the evidence format ids that the verifier supports */
+recv(VERIFIER_SOCKET_FD, evidence_format_ids, evidence_format_id_length, 0);
+
+/* Get registered attester format ids and find a common format */
+oe_get_registered_attester_format_ids(*format_ids, &format_ids_length);
+for (size_t m = 0; m < format_ids_length; m++)
+{
+    for (size_t n = 0; n < evidence_format_id_length; n++)
+    if (format_ids[m] == evidence_format_ids[n])
+    {
+        common_format_id = format_ids[m];
+        break;
+    }
+}
```

Comment:
I feel like this adds to the complexity of this document unnecessarily. This point of this sample code is just to show how one might use the attestation plugin API.

How does this code help the reader understand that? The sample code should detract from the purpose of the doc.

Reply:
I'd like to show at least one reasonable use of the query api. There is no other plugin sample available.

---
In reply to: [362997414](https://github.com/openenclave/openenclave/pull/2402#discussion_r362997414) [](ancestors = 362997414)